pythonopensubtitles/opensubtitles.py

```diff
@@ -6,6 +11,36 @@
     from .settings import Settings
 
 
+def decompress_utf_8(data):
+    """
+    Convert a base64-compressed subtitles file with utf-8 encoding
+    back to a string.
+    """
+    decompressed = None
+    try:
+        decompressed = zlib.decompress(base64.b64decode(data),
+                                       16 + zlib.MAX_WBITS).decode('utf-8')
+    except UnicodeDecodeError as e:
+        print(e)
+
+    return decompressed
+
+
+def decompress_latin(data):
```

Comment:
I think you wanted to say: `decompress_latin1`

Reply:
Oops, very true! Will be obsolete though with the move to a new, combined function.